Task 8/script.js

```diff
@@ -0,0 +1,33 @@
+/* ------------------------------ TASK 8 --------------------------------------------
+Sukurkite konstruktoriaus funkciją "Calculator" (naudokite ES5), kuri gebės sukurti objektus su 4 metodais:
+sum(a, b) - priima du skaičius ir grąžina jų sumą.
+subtraction(a, b) - priima du skaičius ir grąžina jų skirtumą.
+multiplication(a, b) - priima du skaičius ir grąžina jų daugybos rezultatą;
+division(a, b) - priima du skaičius ir grąžina jų dalybos rezultatą;
+------------------------------------------------------------------------------------ */
+
+
+
+function Calculator(a, b) {
+    this.a = a;
```

Comment:
There is no need to get `a` and `b` as an arguments, you will pass them to methods.